src/index.html

```diff
@@ -9,11 +9,31 @@
           maximum-scale=1.0,
           minimum-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Document</title>
+  <title>AirBnB search bar</title>
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-
+  <form method="GET" action="#" class="search-form" data-qa="big">
+    <div class="input__wrapper">
```

Comment:
where is class `input`? this is wrong BEM if you create children element without parent

Reply:
I made class wrapper instead